src/dataaggregator/aggregator/metricPoller.go

```diff
@@ -57,41 +57,36 @@ func (m *MetricPoller) retrieveMetric(app *appmetric.AppMonitor) {
 	appId := app.AppId
 	metricType := app.MetricType
 	endTime := time.Now().UnixNano()
-	startTime := endTime - int64(app.StatWindowSecs)*1000*1000
+	startTime := endTime - int64(app.StatWindow)
```

Comment:
Why are we casting rather than using Duration everywhere until we actually need to cast?